src/m365/teams/commands/cache/cache-remove.ts

```diff
@@ -0,0 +1,113 @@
+import { Cli, Logger } from '../../../../cli';
+import {
+  CommandOption
+} from '../../../../Command';
+import util = require('util');
```

Comment:
This is a [module](https://nodejs.org/api/util.html) that provides acces to some Nodejs utilities. I used this to promisify the exec function.

Reply:
Ah, it wasn't clear to me that it's a part of Node. Shall we change its import syntax to `import * as util from 'util'` so that it's consistent with the rest?